src/main/java/edu/greenblitz/tobyDetermined/subsystems/LED.java

```diff
@@ -31,45 +30,51 @@ public static void init(){
         }
     }
 
-    public void setColor(Color color) {
```

Comment:
I'd like to hear about why you decided to remove this. I saw that you use `setColor(RobotMap.LED.SEctions.ALL)` a few times and I'd honestly expect a method called `setColor()` to apply that color to the entire given range whenever a range is not specified. Maybe I'm missing something about the way we use LEDs but it seems very convenient to me.